src/lib.rs

```diff
@@ -161,25 +163,19 @@ pub fn run_wasm_with_css(css: &str) {
 
     // build wasm example via cargo
     let cargo = env::var("CARGO").unwrap_or_else(|_| "cargo".to_string());
-    let project_root = Path::new(&std::env::var("CARGO_MANIFEST_DIR").unwrap())
-        .ancestors()
-        .nth(1)
-        .unwrap()
-        .to_path_buf();
-    let mut cargo_args = vec![
```

Comment:
I would prefer to keep the vec constructed in a single go like this rather than using `.extend`